GitUI/UserControls/RevisionGrid/FilterInfo.cs

```diff
@@ -194,6 +194,11 @@ public bool HasRevisionFilter
                 || ShowSimplifyByDecoration;
         }
 
+        /// <summary>
+        /// The hash of the last revision to display.
+        /// </summary>
+        public string LastRevisionToDisplayHash { get; internal set; }
```

Comment:
What does "the last revision to display" mean?

Reply:
This is the oldest commit (so the last one at the bottom) that will be displayed. So we use the commit hash to query history only until this commit.

Reply:
Please add this to the xml-doc.